system-int-test/src/cmr/system_int_test/system.clj

```diff
@@ -41,7 +41,7 @@
              ;; A map of the components (echo, elastic, db, and message queue) to whether they are
              ;; in-memory or external
              :component-type-map component-type-map}]
-    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs :ingest])))
+    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :metadata-db :urs :ingest])))
```

Comment:
This looks like the sort of change that might have taken a while to figure out why something wasn't working ... :-/